CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 0.8.2 (Next)
 
+* [#232](https://github.com/mongoid/mongoid-history/pull/232): Bug/187 track changes from embedded documents (not deeply nested) - [@Startouf](https://github.com/Startouf).
```

Comment:
By convention we just do `[#232], [#187]: ...`, but I won't hold the CR for this.